lua/pears/input.lua

```diff
@@ -56,11 +56,11 @@ function Input:_destroy_context(context)
   context:destroy()
 end
 
-function Input:expand(char)
+function Input:expand(char, enter_pressed)
```

Comment:
This might be useful to abstract into a meta-key argument instead of just an entered pressed boolean. That way we can target different types of keys that aren't input characters. Which will also allow rules for other meta keys.

Reply:
Sounds good to me, but I'm not sure about using "meta" in the name of this argument since the meta key is a specific key on the keyboard. Maybe `virtual_key`, or something like that?

Also, do you have an idea for how you would want to represent this value?

Reply:
Meta was probably not the right term to use. Virtual key works fine for me. I think we could just make a table with constants in them. I'm not a fan of magic strings so something like this:

```lua
local M.VirtualKey = {
  ENTER = "enter",
  SPACE = "space"
}
```

This will only have a couple values, but it will scale if we need more "virtual keys".

Reply:
I pushed a commit that addresses this and your other comment. Let me know if this is what you were thinking.

Reply:
This all looks good. I'm nervous about some edge cases, but I think we can address those as they come up!